doc/manuals/admin/management_api.md

```diff
@@ -28,3 +29,67 @@ curl --request PUT <host>:<port>/log/trace/t1-t2,t3-t4
 'PUT-requests' overwrite the previous log settings. So, in order to ADD
 a trace level, a GET /log/trace must be issued first and after that the
 complete trace string to be sent in the PUT request can be assembled.
+
+
+## Semaphores
+Another useful (especially if the broker stops responding correctly) REST API is
+the semaphore listing offered:
+
+
+```
+curl <host>:<port>/admin/sem
+```
+
+The response is a listing of information of all the broker's semaphores:
+
+```
+{
+    "alarmMgr": {
+        "status": "free"
+    },
+    "connectionContext": {
+        "status": "free"
+    },
+    "connectionSubContext": {
+        "status": "free"
+    },
+    "dbConnection": {
+        "status": "free"
+    },
+    "dbConnectionPool": {
+        "status": "free"
+    },
+    "logMsg": {
+        "status": "free"
+    },
+    "request": {
+        "status": "free"
+    },
+    "subCache": {
+        "status": "free"
+    },
+    "timeStat": {
+        "status": "free"
+    },
+    "transaction": {
+        "status": "free"
+    }
+}
+```
+
+Short explanation of the semaphores:
+* **alarmMgr**, protects the data of the Alarm Manager 
+* **connectionContext**, protects the curl context for sending HTTP notifications/forwarded messages
+* **connectionSubContext**, protects the curl endpoint when sending HTTP notifications/forwarded messages
```

Comment:
Maybe the logic behind this "connectionSubContext" field should be explained, e.g. "Note that the value for this field is actually an aggregate of the individual semaphores used for each connection (per endpoint). In particular, if at least one of the semaphores is taken, then its value is `taken` (`free` otherwise).